src/App/root_reducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import login from '../Login/reducer';
```

Comment:
Agree with @jessemao it's not pretty to look. But good point about the downside of being confused. I prefer to keep it as is.